presubmit.py

```diff
@@ -0,0 +1,27 @@
+import os
+from subprocess import check_output, PIPE
+
+def run_test(test_name, command):
+    termCommand = command + ' ' + test_name
+    testName = test_name
+    numDots = 37 - len(testName)
+    dots = ""
+    i = 0
+    while i < numDots:
+        dots+="."
+        i = i + 1
```

Comment:
instead of doing a while loop for the numDots, you can multiply a string in python by a number. For example you can do:  
`
variable = 'hello' * 2            //this is equivalent to saying variable = 'hellohello'

`
Try to implement this rather than using a while loop